_data/task.yml

```
@@ -67,6 +67,15 @@ websites:
       img: rememberthemilk.png
       tfa: No
 
+    - name: Standard Notes
+      url: https://standardnotes.org
+      img: standardnotes.png
+      tfa: Yes
+      software: Yes
+      doc: https://standardnotes.org/help/25/how-do-i-enable-two-factor-authentication-for-my-account
+      exceptions:
+          text: "Requieres Standard Notes extended or self-hosted server."
```

What do you think about this?
```suggestion
          text: "2FA only available for users with Standard Notes Extended or self-hosted server."
```